### PR TITLE
[FIX] point_of_sale: create only one picking when validating order (i…

### PR DESCRIPTION
**Pull request by trj-odoo**
…f the option is activated)

Description of the issue/feature this PR addresses:
With this following configuration: restaurant (with floor management) and the `update quantity in stock` setting set to `real time`, whenever an order is processed (syncing with the database or validating an order), a picking is always created (if there are lines containing storable product).

Current behavior before PR:
Multiple pickings can be created for one order
Desired behavior after PR is merged:
Only one picking is created

### CODE REVIEW

File: addons/point_of_sale/models/pos_order.py [hunk 1]
```diff
@@ -133,13 +133,13 @@ def _process_order(self, order, draft, existing_order):
         if not draft:
             try:
                 pos_order.action_pos_order_paid()
+                pos_order._create_order_picking()
```
Comment (pimodoo) [inline addons/point_of_sale/models/pos_order.py:136]: il ne faut pas le mettre dans le try

File: addons/point_of_sale/models/pos_order.py [hunk 2]
```diff
@@ -131,6 +131,7 @@ def _process_order(self, order, draft, existing_order):
         self._process_payment_lines(order, pos_order, pos_session, draft)
 
         if not draft:
+            pos_order._create_order_picking()
```
Comment (pimodoo) [inline addons/point_of_sale/models/pos_order.py:134]: :) moi je changerais pas l'ordre :D

Comment (trj-odoo) [reply addons/point_of_sale/models/pos_order.py]: Je vois pas de quoi tu parles :eyes:

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**